openshift_images/managing_images/using-image-pull-secrets.adoc

```diff
@@ -6,14 +6,35 @@ include::modules/common-attributes.adoc[]
 toc::[]
 
 If you are using {product-title}'s internal registry and are pulling from
-imagestreams located in the same project, then your Pod's service account should
+imagestreams located in the same project, then your pod's service account should
 already have the correct permissions and no additional action should be
 required.
 
 However, for other scenarios, such as referencing images across {product-title}
 projects or from secured registries, then additional configuration steps are
 required.
 
+You can obtain the image pull secret, `pullSecret`, from the link:https://cloud.redhat.com/openshift/install/pull-secret[Pull Secret] page on the {cloud-redhat-com} site.
+
+You use this pull secret to authenticate with the services that are provided by the included authorities, including link:quay.io[Quay.io] and link:registry.redhat.io[registry.redhat.io], which serves the container images for {product-title} components.
```

Comment:
s/serves/serve (unless I'm misunderstanding and that's not really plural :slightly_smiling_face:)